docs/README-dev.md

```diff
@@ -58,11 +58,15 @@ Because of the above, include paths in source should always feature the full pat
 Note that the recommended way of building the project is to use separate build directory, where CMake will generate all targets. In this way, you can keep a clean source directory. The following shell commands illustrate this usage:
 
 ```sh
-  cd <colobot_repository>
+  git clone https://github.com/colobot/colobot.git
+  cd colobot
+  git submodule update --init
   mkdir build
-  cmake ../
+  cd build
+  # You can't compile translations unless you have python2
+  cmake -D TRANSLATIONS=OFF -D COLOBOT_DEVELOPMENT_MODE=ON ..
   make
-  bin/colobot -datadir ../data
+  ./colobot -datadir data/
```

Comment:
`-datadir` should be unnecessary if you run the game from a directory which contains the data directory.

Reply:
Incorrect. I tried running `./colobot` without `-datadir data/` and got:
```
[WARN]: Zenity not available, will fallback to console users dialogs.
[INFO]: Colobot: Gold Edition 0.2.1.0+alpha-git-dev~rca4049b8 starting
[INFO]: Creating CApplication
[ERROR]: Data directory '/usr/local/share/games/colobot' doesn't exist or is not a directory
ERROR: Could not read from data directory:
'/usr/local/share/games/colobot'
Please check your installation, or supply a valid data directory by -datadir option.
Press ENTER to continue
[INFO]: Didn't run main loop. Exiting with code 1
```

What you are describing only works with USE_RELATIVE_PATHS=ON or PORTABLE=ON, but both are OFF by default. COLOBOT_DEVELOPMENT_MODE=ON does not enable either of the two options